src/containers/collaborations/components/CreateCollaborationModal.js

```diff
@@ -115,23 +99,19 @@ const CreateCollaborationModal = ({ onClose } :Props) => {
   };
 
   const handleOnClickPrimary = () => {
-    if (isNonEmptyString(modalState.name) && isNonEmptyString(modalState.title)) {
+    if (isNonEmptyString(modalState.title)) {
+      const name = modalState.title.replace(/\W/g, '');
       dispatch(
         createNewCollaboration({
           description: modalState.description,
-          name: modalState.name,
+          name,
           organizationIds: collaborationOrganizations,
           title: modalState.title,
         })
       );
     }
-    else {
-      if (!isNonEmptyString(modalState.name)) {
-        modalDispatch({ type: NAME_IS_VALID, value: false });
-      }
-      if (!isNonEmptyString(modalState.title)) {
-        modalDispatch({ type: TITLE_IS_VALID, value: false });
-      }
+    else if (!isNonEmptyString(modalState.title)) {
```

Comment:
I believe this can be just an `else`.

```suggestion
    else {
```